src/browser/provider/plugin-host.js

```diff
@@ -0,0 +1,64 @@
+import { assignIn } from 'lodash';
+
+
+const helpers = Symbol();
+
+
+export default class BrowserProviderPluginHost {
+    constructor (providerObject, helpersObject) {
+        assignIn(this, providerObject);
+
+        this[helpers] = helpersObject;
+    }
+
+    // Helper
+    runInitScript (id, func) {
+        return this[helpers].runInitScript(id, `(${func.toString()})()`);
```

Comment:
Not quite safe way to do that, because `func` code can be augmented by Babel artifacts. It's better to pass string here.